webui/react/src/components/InteractiveTable.tsx

```diff
@@ -363,17 +363,20 @@ const InteractiveTable: InteractiveTable = ({
 
   const handleChange = useCallback(
     (tablePagination, tableFilters, tableSorter): void => {
-      if (Array.isArray(tableSorter)) return;
 
-      const { columnKey, order } = tableSorter as SorterResult<unknown>;
-      if (!columnKey || !settings.columns.find((col) => columnDefs[col]?.key === columnKey)) return;
+      if (Array.isArray(tableSorter)) return;
 
-      const newSettings = {
-        sortDesc: order === 'descend',
-        sortKey: columnKey,
+      const newSettings: Partial<InteractiveTableSettings> = {
         tableLimit: tablePagination.pageSize,
         tableOffset: (tablePagination.current - 1) * tablePagination.pageSize,
       };
+
+      const { columnKey, order } = tableSorter as SorterResult<unknown>;
+      if (columnKey && settings.columns.find((col) => columnDefs[col]?.key === columnKey)) {
```

Comment:
`settings.columns` is the list of displayed columns, which might not include all columns.

so if we were sorting on a column, and then hide that column, this code would cause it to no longer sort by that column. 

this PR doesn't change the functionality, but it at least got me wondering it that is the right thing. what do you think?

Reply:
I think what makes most sense to me is to keep sorted on selected column, even that column got hidden.